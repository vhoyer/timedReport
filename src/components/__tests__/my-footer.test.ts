import { describe, it, expect } from 'vitest';
import { lazylet } from '@vhoyer/lazy-let';
import { render } from '@testing-library/vue';
import MyFooter from '../my-footer.vue';

describe('Components > Footer', () => {
  const $ = lazylet({
    wrapper: () => render(MyFooter),
  });

  it('renders correctly', () => {
    expect($.wrapper.container).toMatchSnapshot();
  });
});
