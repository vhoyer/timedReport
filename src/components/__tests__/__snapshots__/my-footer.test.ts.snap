// Vitest Snapshot v1

exports[`Components > Footer > renders correctly 1`] = `
<div>
  <footer
    class="pt-4 pb-4"
    data-v-53855059=""
  >
    <div
      class="container"
      data-v-53855059=""
    >
      <div
        class="row"
        data-v-53855059=""
      >
        <small
          class="col text-muted"
          data-hj-whitelist=""
          data-v-53855059=""
        >
           TimedReport!™ — next 
        </small>
        <small
          class="col text-right text-muted"
          data-v-53855059=""
        >
           Vinícius Hoyer 
        </small>
      </div>
    </div>
  </footer>
</div>
`;
